test/unit/interface.cpp

```diff
@@ -144,15 +144,15 @@ TEST(Interface, ParametersSet)
 {
     int td = 3;
     int k = 5;
-    tapkee::ParametersSet pg = tapkee::kwargs[target_dimension = td, num_neighbors = k];
+    tapkee::ParametersSet pg = tapkee::kwargs[(target_dimension = td, num_neighbors = k)];
     ASSERT_EQ(static_cast<int>(pg[target_dimension]), td);
     ASSERT_EQ(static_cast<int>(pg[num_neighbors]), k);
 }
 
 TEST(Interface, OneParameterParametersSet)
 {
     int td = 3;
-    tapkee::ParametersSet pg = tapkee::kwargs[target_dimension = td];
+    tapkee::ParametersSet pg = tapkee::kwargs[(target_dimension = td)];
```

Comment:
Oeps, I think I overlooked when changing this one ':-) without parentheses there should still be no warning in this one. I will modify it back.